ledger-tool/src/output.rs

```diff
@@ -0,0 +1,51 @@
+use {
+    serde::Serialize,
+    solana_cli_output::{QuietDisplay, VerboseDisplay},
+    std::fmt::{Display, Formatter, Result},
+};
+
+#[derive(Serialize, Debug, Default)]
+pub struct SlotInfo {
+    pub start: u64,
+    pub end: u64,
+    pub total: usize,
+    #[serde(rename = "fromLast", skip_serializing_if = "Option::is_none")]
+    pub from_last: Option<u64>,
+}
+
+impl Display for SlotInfo {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result {
+        writeln!(f, "Start: {}", &self.start)?;
+        writeln!(f, "End: {}", &self.end)?;
+        writeln!(f, "Total: {}", &self.total)?;
+        if let Some(from_last) = &self.from_last {
+            writeln!(f, "From Last: {}", from_last)?;
+        }
+        Ok(())
+    }
+}
+
+#[derive(Serialize, Debug, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct SlotBounds {
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub total_slots: Option<SlotInfo>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub rooted_slots: Option<SlotInfo>,
+}
```

Comment:
I'm honestly not sure of which of these is preferred for the empty ledger case:
```
{}
```
vs.
```
{
  "totalSlot": null,
  "rootedSlots": null
}
```
Will circle back on this unless Tyera chimes in with any opinions; I was probably going to look at some of our RPC methods to see what convention we use there

Reply:
I actually think something like this would be more clear (using my proposed field-name changes):
```
{
    slots: {
        total: 0,
        first: null,
        last: null,
        num_after_last_root: 0
    },
    roots: {
        total: 0,
        first: null,
        last: null
    }
}
```
Will require making a couple more `SlotInfo` fields Options.

Reply:
+1 for above